.env.template

```diff
@@ -0,0 +1 @@
+GLOBAL_ENVIRONMENT="DEV" / "LIVE"
```

Comment:
just googled how this thing should look. So it would be preferable to be more specific about the name of this variable. You can call it `BACK_ENV` since it is for the backend. The word `GLOBAL` means too much in this context. Also, instead of `DEV` and `LIVE`, use `DEV` and `PROD` from `development` and `production`